packages/utilities/psammead-locales/moment/ps.js

```diff
@@ -27,11 +27,10 @@ moment.defineLocale('ps', {
   months: pashtoGregorianMonths,
   monthsShort: pashtoGregorianMonths,
   relativeTime: {
-    past: '%s مخکې', // %s 'ago'
-    s: 'یو څو ثانیې', // 'a few seconds'
-    m: 'یوه دقیقه', // 'a minute'
+    past: '%s وړاندې', // %s 'ago'
+    m: '۱ دقیقه', // '1 minute'
```

Comment:
What's the significance of the comma here and on L33?

Reply:
In a text editor that comma terminates the line correctly. RTL seems to be playing tricks on Github, because that `1` doesn't exist either.